charts/generic-data-analytics-extractor/README.md

```diff
@@ -0,0 +1,39 @@
+# generic-data-analytics-extractor chart
+
+This chart creates a standard cron job to extract **all data** from a database and send it to an s3 bucket for analysis.
+
+This is the final of three steps to set up daily transfers to the analytical platform as detailed in [this guide](https://dsdmoj.atlassian.net/wiki/spaces/PPDE/pages/3297050829/Steps+to+set+up+daily+transfers+to+the+analytical+platform).
+
+Before using this chart you will need to:
+- [prepare the name space](https://dsdmoj.atlassian.net/wiki/spaces/PPDE/pages/3297050829/Steps+to+set+up+daily+transfers+to+the+analytical+platform#Prepare-namespace).
+- [register data with the analytical platform](https://dsdmoj.atlassian.net/wiki/spaces/PPDE/pages/3297050829/Steps+to+set+up+daily+transfers+to+the+analytical+platform#Register-data-with-the-analytical-platform).
+
+By default we have disabled the cron job due to the Production environment needing a [Data protection impact assessment](https://dsdmoj.atlassian.net/wiki/spaces/PPDE/pages/3491823875/Data+protection+impact+assessments+for+microservice+data). To override this, set `enabled: true` in you applications helm chart `values.yaml`:
```

Comment:
maybe mention production data rather than environment here?

Reply:
Sorted in latest commit